javascript/0162-find-peak-element.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Suggestion: Add Time O() | Space O() complexity

Reply:
Great suggestion. I made the changes